docs/data-management.rst

```diff
@@ -508,6 +510,13 @@ Step-by-Step
     duplicates, the link might be deleted during the subsequent
     move of that duplicate to another group.
 
+.. _Shares:
+
+Shares are deprecated
+~~~~~~~~~~~~~~~~~~~~~
+
+Duplicating and moving the data into another group as described in :ref:`Duplicate feature<Duplicatecli>` is the recommended way you cab share objects in OMERO with users who are not members of the group where the data are located. Please **do not use the** ``Shares``  tab |image6| above the left-hand pane in OMERO.web to create new Shares as this feature is deprecated and not supported and tested any more. The deprecation warning will be added to OMERO.web in future.
```

Comment:
Typo "you cab share"

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/19/commits/ac12cba09b90384a60d032851fae174bd2fc1ea9